src/js/game/Game.js

```diff
@@ -63,17 +75,21 @@ export default class Game {
       Matter.Composite.add(this.engine.world, [wall.body]);
     });
     // Sand
-    this.map.coords.sands.forEach((s) => {
-      const sand = new Sand(s.x, s.y, s.w, s.h);
-      this.sands.push(sand);
-      this.viewport.addChild(sand.sprite);
-    });
+    if (this.map.coords.sands) {
```

Comment:
Behöver den wrappas i en if? Om det inte finns några sands då kommer väl koden inte köras ändå?

Reply:
Tyvärr inte! Lade in den för att pixi krashade när jag försökte gå till nästa bana. Med de två så borde allting fungera som vanligt både med och utan hinder 👍🏻